Stack/Opc.Ua.Core/Types/Utils/Utils.cs

```diff
@@ -1535,353 +1535,19 @@ public static object Clone(object value)
-            // copy ServerStatusDataType.
-            {
-                ServerStatusDataType castedObject = value as ServerStatusDataType;
-                if (castedObject != null)
-                {
-                    return castedObject.MemberwiseClone();
-                }
-            }
-            // copy BuildInfo.
-            {
-                BuildInfo castedObject = value as BuildInfo;
-                if (castedObject != null)
-                {
-                    return castedObject.MemberwiseClone();
-                }
-            }
-            // copy X509IdentityToken.
-            {
-                X509IdentityToken castedObject = value as X509IdentityToken;
-                if (castedObject != null)
-                {
-                    return castedObject.MemberwiseClone();
-                }
-            }
-            // copy Opc.Ua.Range.
-            {
-                Opc.Ua.Range castedObject = value as Opc.Ua.Range;
-                if (castedObject != null)
-                {
-                    return castedObject.MemberwiseClone();
-                }
-            }
-            // copy Opc.Ua.EUInformation
-            {
-                Opc.Ua.EUInformation castedObject = value as Opc.Ua.EUInformation;
-                if (castedObject != null)
-                {
-                    return castedObject.MemberwiseClone();
-                }
-            }
-            // copy Opc.Ua.WriteValueCollection
-            {
-                Opc.Ua.WriteValueCollection castedObject = value as Opc.Ua.WriteValueCollection;
-                if (castedObject != null)
-                {
-                    return castedObject.MemberwiseClone();
-                }
-            }
-            // copy Opc.Ua.WriteValue
-            {
-                Opc.Ua.WriteValue castedObject = value as Opc.Ua.WriteValue;
-                if (castedObject != null)
-                {
-                    return castedObject.MemberwiseClone();
-                }
-            }
-            // copy Opc.Ua.DataValue
-            {
-                Opc.Ua.DataValue castedObject = value as Opc.Ua.DataValue;
-                if (castedObject != null)
-                {
-                    return castedObject.MemberwiseClone();
-                }
-            }
-            // copy Opc.Ua.ExpandedNodeId
-            {
-                ExpandedNodeId castedObject = value as ExpandedNodeId;
-                if (castedObject != null)
-                {
-                    return castedObject.MemberwiseClone();
-                }
-            }
-            // copy Opc.Ua.TimeZoneDataType
-            {
-                TimeZoneDataType castedObject = value as TimeZoneDataType;
-                if (castedObject != null)
-                {
-                    return castedObject.MemberwiseClone();
-                }
-            }
-            // copy Opc.Ua.LiteralOperand
+            dynamic dynamicValue = value;
+            try
```

Comment:
try solution without dynamic keyword

Reply:
I reverted to the original solution. - But still it is not a correct one. I'd like to discuss that with the workgroup on the next meeting.